data/baseline_situations_generated/traffic_situation_02.json

```diff
@@ -1,68 +1,47 @@
 {
     "title": "CR-GW",
-    "inputFileName": "baseline_situation_02_1_ts.json",
-    "commonVector": 600.0,
     "ownShip": {
         "static": {
-            "id": "0216291f-1ce2-4e76-b752-7cf93e48f6ab",
+            "id": "6983e2ad-4f17-414d-9c07-5c12214926cc",
             "length": 122.0,
             "width": 20.0,
             "height": 8.0,
-            "speedMax": 17.0,
             "mmsi": 257847600,
             "name": "BASTO VI",
-            "shipType": "Passenger"
+            "shipType": "Passenger",
+            "speed_max": 17
         },
         "initial": {
             "position": {
-                "north": 0.0,
-                "east": 0.0,
                 "latitude": 1.02561566,
-                "longitude": 0.18309645
+                "longitude": 0.18309645,
+                "east": 0,
+                "north": 0
             },
             "sog": 5.14,
-            "cog": 0.0
+            "cog": 0.0,
+            "navStatus": "Under way using engine"
         },
         "waypoints": [
             {
                 "position": {
-                    "north": 0.0,
-                    "east": 0.0,
                     "latitude": 1.02561566,
-                    "longitude": 0.18309645
+                    "longitude": 0.18309645,
+                    "east": 0,
+                    "north": 0
                 }
             },
             {
                 "position": {
-                    "north": 18504.0,
-                    "east": 0.0,
                     "latitude": 1.02851496,
-                    "longitude": 0.18309645
+                    "longitude": 0.18309645,
+                    "north": 18504.0,
+                    "east": 0.0
                 }
             }
         ]
     },
-    "targetShips": [
-        {
-            "static": {
-                "id": "5b1dbefb-9c43-4edb-ade7-e3f90620bb41",
-                "length": 122.0,
-                "width": 20.0,
-                "height": 8.0,
-                "speedMax": 17.0,
-                "name": "BASTO",
-                "shipType": "Passenger"
-            },
-            "initial": {
-                "position": {
-                    "north": 5774.0,
-                    "east": 2102.0,
-                    "latitude": 1.02652036,
-                    "longitude": 0.18373041
-                },
-                "sog": 4.1,
-                "cog": 3.934
-            }
-        }
-    ]
+    "targetShips": [],
```

Comment:
is it correct that targetShips became an empty list, compared to the former version?  (comment applies to all 55 situations. Picked this specific situation just randomly)

Reply:
Very strange... will need to investigate this further tomorrow  perhaps together with @tomarnepedersen

Reply:
should be fixed now